pinot-api/src/main/java/com/linkedin/pinot/client/ExternalViewReader.java

```diff
@@ -114,4 +120,27 @@ public ExternalViewReader(ZkClient zkClient) {
     return tableToBrokersMap;
   }
 
+  private static byte[] unpackZnodeIfNecessary(byte[] znodeContents) {
```

Comment:
The thing is that if we do that, it pulls in all of Helix into the Pinot client.